packages/react/src/components/dropdown-list/dropdown-list.test.tsx

```diff
@@ -182,6 +182,21 @@ describe('Dropdown list', () => {
         });
     });
 
+    describe('icon is handled', () => {
+        test('the icon is set according to the iconName prop', () => {
+            const wrapper = shallow(<DropdownList options={provinces} iconName="menu" />);
+
+            expect(getByTestId(wrapper, 'textboxIcon').prop('name')).toBe('menu');
+        });
+
+        test('the icon is not set when the iconName prop is undefined', () => {
+            const wrapper = shallow(<DropdownList options={provinces} />);
+
+            const iconComponent = getByTestId(wrapper, 'textboxIcon');
+            expect(iconComponent).toBeTruthy();
```

Comment:
Pas la bonne condition.

```suggestion
            expect(iconComponent.exists()).toBe(false);
```